book/MovingToTheGPU.html

```diff
@@ -113,31 +113,30 @@
 
   wgpu also has [native bindings in other languages](https://github.com/gfx-rs/wgpu-native).
 
-* I use [*winit*](https://docs.rs/winit/latest/winit/) which is a portable windowing library. I use
-  it to view the rendered image in real-time and to make the example code interactive.
+* I use [*winit*](https://docs.rs/winit/latest/winit/) which is a portable windowing library. Used
+  to view the rendered image in real-time and make the example code interactive.
 
 * For ease of Rust development I use [*anyhow*](https://docs.rs/anyhow/latest/anyhow/) and
   [*bytemuck*](https://docs.rs/bytemuck/latest/bytemuck/). *anyhow* is a popular error handling
   utility and integrates seamlessly. *bytemuck* provides a safe abstraction for the equivalent of
-  `reinterpret_cast` in C++, which normally requires [`unsafe`][rust-unsafe] Rust. I use it to
+  `reinterpret_cast` in C++, which normally requires [`unsafe`][rust-unsafe] Rust. It's used to
   bridge CPU data types with their GPU equivalents.
 
 * Lastly, I use [*pollster*](https://docs.rs/pollster/latest/pollster/) to execute asynchronous
   wgpu API functions (which is only called from a single line).
```

Comment:
I removed some redundant "I use" verbiage, but we could go further and initially state "These are the dependencies used". Each dependency listed then can have justification for it's use.

Reply:
I like the passive voice that you went with.